api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -372,9 +376,14 @@ const (
 )
 
 // PlatformType is a specific supported infrastructure provider.
-// +kubebuilder:validation:Enum="";BareMetal;None;VSphere;Nutanix
+// +kubebuilder:validation:Enum="";BareMetal;None;VSphere;Nutanix;External
 type PlatformType string
 
+// ExternalSpec contains a specific supported platform name.
+type ExternalSpec struct {
```

Comment:
```suggestion
// ExternalPlatformSpec stores configuration related to external platform.
type ExternalPlatformSpec struct {
```
I think it could be a bit more clear

Reply:
Copy-paste from here
https://github.com/openshift/api/blob/edd317298780d562af7d5d979b237d14fc299785/config/v1/types_infrastructure.go#L232-L241